packages/babel-plugin-react-i18n/src/BabelPluginReactI18n.ts

```diff
@@ -0,0 +1,97 @@
+import {TemplateBuilder} from '@babel/template';
+import * as Types from '@babel/types';
+import {NodePath} from '@babel/traverse';
+import stringHash from 'string-hash';
+import path from 'path';
+import fs from 'fs';
+
+interface State {
+  importer(importArg: Types.ImportDeclaration): void;
+}
+
+const WITH_I18N_DECORATOR_NAME = 'withI18n';
+
+export default function injectWithI18nArguments({
+  types: t,
+  template,
+}: {
+  types: typeof Types;
+  template: TemplateBuilder<Types.ImportDeclaration | Types.ObjectExpression>;
+}) {
+  const fallbackTranslationsImport = template(
+    `import enTranslations from './translations/en.json';`,
```

Comment:
I think there is an API on path to get a unique binding name that we probably want to use instead